src/MenuList.js

```diff
@@ -38,7 +38,7 @@ const MenuList = ({ children, className }) => (
  */
 MenuList.propTypes = {
     className: propTypes.string,
-    children: propTypes.element.isRequired,
+    children: propTypes.node.isRequired,
```

Comment:
MenuList accepts an array (or at least it receives an array in the storybook demo), which element does not allow (it expects a single react element), but node does.